_config.dev.yml

```diff
@@ -9,13 +9,16 @@ future: true
 
 # Don't generate this stuff
 exclude:
-  - README.md
-  - CNAME
```

Comment:
is the "CNAME" need to be served?

Reply:
Good question! This is only needed for GitHub Pages. I'll remove the file.